lib/Onnxifi/onnxifiGlow.cpp

```diff
@@ -302,12 +302,32 @@ ONNXIFI_PUBLIC ONNXIFI_CHECK_RESULT onnxStatus ONNXIFI_ABI onnxInitGraph(
   return ONNXIFI_STATUS_SUCCESS;
 }
 
+static bool verifyDescriptors(uint32_t count,
+                              const onnxTensorDescriptorV1 *descriptors) {
+  for (unsigned i = 0; i < count; i++) {
+    const auto &descriptor = descriptors[i];
+    /*if (descriptor.tag != ONNXIFI_TAG_TENSOR_DESCRIPTOR_V1) {
```

Comment:
@yinghai seems like C2 does not set the tag here properly on onnxSetGraphIO (i have to comment out this piece of code in order to process descriptors). descriotpr.tag equals to 0 at this point.

Reply:
https://github.com/pytorch/pytorch/pull/10502 fixed it and is already merged.

Reply:
Nice! Will update it!